hugo/content/guides/multiple-languages.md

```diff
@@ -0,0 +1,454 @@
+            "id": "multiple-languages-definition",
+            "aliases": [
+                "Multiple Languages Definition",
+                "multiple-languages-definition"
+            ],
+            "extensions": [".who"],
+            "configuration": "./language-configuration.json"
+        }, {
+            "id": "multiple-languages-implementation",
+            "aliases": [
+                "Multiple Languages Implementation",
+                "multiple-languages-implementation"
+            ],
+            "extensions": [".hello"],
+            "configuration": "./language-configuration.json"
+        }
+    ],
+    "grammars": [
+        {
+            "language": "multiple-languages-configuration",
+            "scopeName": "source.multiple-languages-configuration",
+            "path": "./syntaxes/multiple-languages-configuration.tmLanguage.json"
+        },
+        {
+            "language": "multiple-languages-definition",
+            "scopeName": "source.multiple-languages-definition",
+            "path": "./syntaxes/multiple-languages-definition.tmLanguage.json"
+        },
+        {
+            "language": "multiple-languages-implementation",
+            "scopeName": "source.multiple-languages-implementation",
+            "path": "./syntaxes/multiple-languages-implementation.tmLanguage.json"
+        }
+    ]
+},  
+```
+
+#### File `src/extension/main.ts`
+
+And here is the extension file and how the file watcher in it worked before the change:
+
+```ts
+const fileSystemWatcher = vscode.workspace.createFileSystemWatcher('**/*.hello');
+context.subscriptions.push(fileSystemWatcher);
+
+// Options to control the language client
+const clientOptions: LanguageClientOptions = {
+    documentSelector: [{ scheme: 'file', language: 'multiple-languages' }],
+    synchronize: {
+        // Notify the server about file changes to files contained in the workspace
+        fileEvents: fileSystemWatcher
+    }
+};
+```
+
+After the change, it should look like this (the language IDs should be the same as they are in the Langium configuration):
+
+```ts
+const fileSystemWatcher = vscode.workspace.createFileSystemWatcher('**/*.(me|who|hello)');
+context.subscriptions.push(fileSystemWatcher);
+
+// Options to control the language client
+const clientOptions: LanguageClientOptions = {
+    documentSelector: [
+        { scheme: 'file', language: 'multiple-languages-configuration' },
+        { scheme: 'file', language: 'multiple-languages-definition' },
+        { scheme: 'file', language: 'multiple-languages-implementation' }
+    ],
+    synchronize: {
+        // Notify the server about file changes to files contained in the workspace
+        fileEvents: fileSystemWatcher
+    }
+};
+```
+
+> Be aware of that the file extensions can change over time, depending on your needs. You should actually use the `LanguageMetaData` of each language service. I have chosen the given implementation because of simplicity. You can do it better!
```

Comment:
```suggestion
> Be aware that the file extensions can change over time, depending on your needs. You should actually use the `LanguageMetaData` of each language service. I have chosen the given implementation because of simplicity. You can do it better!
```